rts/lra/lra-test/lra-test-tck/pom.xml

```diff
@@ -25,6 +25,31 @@
         <lra.tck.consistency.longDelay>20000</lra.tck.consistency.longDelay>
 
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+
+        <it.test>
```

Comment:
Will you include a comment explaining why these tests are skipped and also update the [issue](https://issues.redhat.com/browse/JBTM-3261) to explain what still needs doing.

Reply:
ok